sdks/java/extensions/sql/build.gradle

```diff
@@ -177,16 +177,6 @@ task runPojoExample(type: JavaExec) {
   args = ["--runner=DirectRunner"]
 }
 
-// These tests become flaky when run in parallel at more than 3 threads so run them in a separate task
-task runKafkaTableProviderIT(type: Test) {
-  outputs.upToDateWhen { false }
-  include '**/KafkaTableProvider*IT.class'
-  maxParallelForks 2
-  classpath = project(":sdks:java:extensions:sql").sourceSets.test.runtimeClasspath
-  testClassesDirs = files(project(":sdks:java:extensions:sql").sourceSets.test.output.classesDirs)
-  useJUnit { }
-}
-
```

Comment:
Can we keep the Kafka test in a separate non-parallel task so the other ITs can still run in parallel?

Reply:
I'll replace the inheritance with parameterized tests, there shouldn't be need for a separate Kafka task. The same problem was with Pubsub IT tests so the task would have to receive GCP credentials as well to be able to run these as well.